lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -7,6 +7,10 @@
 #ifndef TEE_API_DEFINES_EXTENSIONS_H
 #define TEE_API_DEFINES_EXTENSIONS_H
 
+/* RSA signatures with MD5 hash */
+#define TEE_ALG_RSASSA_PKCS1_PSS_MGF1_MD5       0x70111930
+#define TEE_ALG_RSAES_PKCS1_OAEP_MGF1_MD5       0x60110230
```

Comment:
This commit is about RSA sign/verify but `TEE_ALG_RSAES_PKCS1_OAEP_MGF1_MD5` is about RSa ciphering. I think related added lines should be removed from this change.

Reply:
I agree, either the commit could be split in two, or the commit description needs to be fixed to mention both signing and cipher (it's a small change so it is acceptable to have both in one commit IMO).

Reply:
Applied: 
- Mention both Sign and Cipher in commit description
- Add a link to the AOSP vts test
- Use "Link:" tag to refer for the commit in the description
@jforissier Please let me know if the Acked-by is still ok to be added

Thank you

Reply:
Absolutely. You don't need the blank line between the `Link:` and `Signed-off-by:` lines though.

Reply:
Applied, Thank you